ci-operator/step-registry/ipi/conf/vsphere/check/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- abhinavdahiya
+- jcpowermac
```

Comment:
I don't think I'll have the capacity to own this section. So maybe just drop me

Reply:
@abhinavdahiya no problem. Started this change months ago finally gotten back to it.